crates/holochain_sqlite/Cargo.toml

```
@@ -39,10 +42,26 @@ tokio_helper = { version = "0.0.1", path = "../tokio_helper" }
 tracing = "0.1.18"
 tracing-futures = "0.2"
 
+rusqlite = { version = "0.24", features = [
+  "blob",        # better integration with blob types (Read, Write, etc)
+  "chrono",      # integration with chrono crate
+  "functions",   # rust scalar / agg / window functions
+  "hooks",       # sqlite notification callbacks
+  "serde_json",  # integration with serde_json crate
+  "url",         # integration with url crate
+  "uuid",        # integration with uuid crate
```

```suggestion
  #"chrono",      # integration with chrono crate
  #"functions",   # rust scalar / agg / window functions
  #"hooks",       # sqlite notification callbacks
  #"serde_json",  # integration with serde_json crate
  #"url",         # integration with url crate
  #"uuid",        # integration with uuid crate
```

I had these all enabled in my spike but we may not need/want them all. `chrono` in particular I found to not be very useful - it stores the dates as text fields, making it hard to do queries on it... we'll probably want to represent timestamps as INT or REAL so we can query.